stdlib/public/core/Sort.swift

```diff
@@ -518,13 +518,13 @@ extension UnsafeMutableBufferPointer {
     at i: Int,
     buffer: UnsafeMutablePointer<Element>,
     by areInIncreasingOrder: (Element, Element) throws -> Bool
-  ) rethrows -> Bool {
```

Comment:
As far as I understand all ABI-public(`public`, `open`, `@usableFromInline`, or `@inlinable`) functions should not change signature to preserve ABI. In my limited understanding we are allowed to change the implementation details bu not the API.

Reply:
Ahaa. That actually makes sense, thank you @LucianoPAlmeida as always ❤️